src/_data/capability-matrix.yml

```diff
@@ -505,6 +575,10 @@ categories:
             l1: 'No'
             l2: ''
             l3: ''
+          - class: mapreduce
+            l1: 'Yes'
```

Comment:
I think the most useful thing for users here is to say "No" and indicate that it is a batch-only runner.

Reply:
done